packages/interface/src/store/actions.js

```diff
@@ -33,11 +33,13 @@ export const setDefaultComplementaryArea = ( scope, area ) => {
 /**
  * Enable the complementary area.
  *
- * @param {string} scope Complementary area scope.
- * @param {string} area  Area identifier.
+ * @param {string}  scope           Complementary area scope.
+ * @param {string}  area            Area identifier.
+ * @param {Object}  options         Options.
+ * @param {boolean} options.persist Whether to persist.
  */
 export const enableComplementaryArea =
-	( scope, area ) =>
+	( scope, area, { persist = true } = {} ) =>
```

Comment:
Alternatively we could add a private action if we don't want to expose this option.